lib/administration/user/src/user_administration.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "irods/irods_query.hpp"
 #include "irods/query_builder.hpp"
+#include <irods/version.hpp>
```

Comment:
Please use quotation marks instead of the angle brackets here:
```suggestion
#include "irods/version.hpp"
```